memory.js

```diff
@@ -0,0 +1,24 @@
+$(function() {
+	var lastClicked;
+
+	$('td').click(function(){
+		if (lastClicked === undefined){
+			// first click
+			lastClicked = this;
+		} else if (this != lastClicked && $(lastClicked).text() === $(this).text()){
```

Comment:
How would you explain what this condition is checking? Try moving it to a method to make it more descriptive.